src/Html/AssetHtmlGenerator.php

```diff
@@ -51,14 +51,21 @@ class AssetHtmlGenerator
     private $htmlBuilder;
 
 
+    /**
+     * @var bool
+     */
+    private $allowCors;
+
+
     /**
      */
     public function __construct (
         AssetsRegistry $registry,
         AssetUrl $assetUrl,
         FileTypeRegistry $fileTypeRegistry,
         bool $isDebug,
-        DependencyMapFactory $dependencyMapFactory
+        DependencyMapFactory $dependencyMapFactory,
+        bool $allowCors
```

Comment:
```suggestion
        bool $allowCors = false
```

You broke nearly all tests, because this here is a breaking change.

Reply:
Yeah sorry. I forgot to mention that I had to leave. So I couldn’t update the Test. Thx for the review anyway.